lib/publiccloud/instance.pm

```diff
@@ -142,6 +142,12 @@ sub _prepare_ssh_cmd {
     $args{username} //= $self->username();
     $args{timeout} //= SSH_TIMEOUT;
 
+
+    # Ensure -o BatchMode=yes is always included in ssh_opts
+    if ($args{ssh_opts} !~ m{-o\s+BatchMode=yes}) {
```

Comment:
Why not to deal with it via https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/publiccloud/ssh_config ? (do I need to check if SAP tests are using this option too?)